tests/accelerators/test_accelerator_connector.py

```diff
@@ -468,10 +470,7 @@ def test_accelerator_gpu():
     assert trainer._device_type == "gpu"
     assert isinstance(trainer.accelerator, GPUAccelerator)
 
-    with pytest.raises(
-        MisconfigurationException, match="You passed `accelerator='gpu'`, but you didn't pass `gpus` to `Trainer`"
-    ):
-        trainer = Trainer(accelerator="gpu")
+    trainer = Trainer(accelerator="gpu")
```

Comment:
Can you add some assertion for this trainer?

```suggestion
    trainer = Trainer(accelerator="gpu")
    assert isinstance(trainer.accelerator, GPUAccelerator)
```